incubator/common/pipelines/default/build-task.yaml

```diff
@@ -32,6 +32,20 @@ spec:
       volumeMounts:
         - mountPath: /var/lib/containers
           name: varlibcontainers
+    - name: validate-collection-is-active
+      securityContext:
+        privileged: true
+      image: smcclem/validate-collection
```

Comment:
We should not be using personal docker images. Can this be included in an existing image that is being used otherwise we will need another formal image created somewhere.

Reply:
Opps, good catch. I already pushed the image to kabanero.

https://hub.docker.com/r/kabanero/validate-collection